tests/render_test.toml

```diff
@@ -3,9 +3,9 @@
 launcher = "Steam"
 
 [match]
-game_mode = "Soccer"
+game_mode = "Soccar"
 game_map_upk = "Stadium_P"
-enable_rendering = true
+enable_rendering = "OffByDefault"
```

Comment:
```suggestion
enable_rendering = "OnByDefault"
```

Reply:
You requested the feature that core still accepts true/false for enable_rendering in tomls, so true/false are valid shorthand for "OnByDefault" and "AlwaysOff"

Reply:
I also set `OffByDefault` because the script sends a request to enable rendering for testing purposes